specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-04-01-preview/kustoPool.json

```diff
@@ -0,0 +1,1676 @@
+        "PSV",
+        "TXT",
+        "RAW",
+        "SINGLEJSON",
+        "AVRO",
+        "TSVE",
+        "PARQUET",
+        "ORC",
+        "APACHEAVRO",
+        "W3CLOGFILE"
+      ],
+      "x-ms-enum": {
+        "name": "iotHubDataFormat",
+        "modelAsString": true
+      }
+    },
+    "EventGridDataFormat": {
+      "description": "The data format of the message. Optionally the data format can be added to each message.",
+      "type": "string",
+      "enum": [
+        "MULTIJSON",
+        "JSON",
+        "CSV",
+        "TSV",
+        "SCSV",
+        "SOHSV",
+        "PSV",
+        "TXT",
+        "RAW",
+        "SINGLEJSON",
+        "AVRO",
+        "TSVE",
+        "PARQUET",
+        "ORC",
+        "APACHEAVRO",
+        "W3CLOGFILE"
+      ],
+      "x-ms-enum": {
+        "name": "eventGridDataFormat",
+        "modelAsString": true
+      }
+    },
+    "BlobStorageEventType": {
+      "description": "The name of blob storage event type to process.",
+      "type": "string",
+      "enum": [
+        "Microsoft.Storage.BlobCreated",
+        "Microsoft.Storage.BlobRenamed"
+      ],
+      "x-ms-enum": {
+        "name": "blobStorageEventType",
+        "modelAsString": true
+      }
+    },
+    "ResourceProvisioningState": {
+      "type": "string",
+      "readOnly": true,
+      "description": "The provisioned state of the resource.",
+      "enum": [
+        "Running",
+        "Creating",
+        "Deleting",
+        "Succeeded",
+        "Failed",
+        "Moving",
+        "Canceled"
+      ],
+      "x-ms-enum": {
+        "name": "ResourceProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "Compression": {
+      "description": "The compression type",
+      "type": "string",
+      "enum": [
+        "None",
+        "GZip"
+      ],
+      "x-ms-enum": {
+        "name": "compression",
+        "modelAsString": true
+      }
+    }
+  },
+  "parameters": {
+    "kustoPoolNameParameter": {
+      "name": "kustoPoolName",
+      "in": "path",
+      "description": "The name of the Kusto pool.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "OperationIdParameter": {
+      "name": "operationId",
+      "in": "path",
+      "description": "The Guid of the operation ID",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "DatabaseNameParameter": {
+      "name": "databaseName",
+      "in": "path",
+      "description": "The name of the database in the Kusto pool.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "DataConnectionNameParameter": {
+      "name": "dataConnectionName",
+      "in": "path",
+      "description": "The name of the data connection.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "securityDefinitions": {
```

Comment:
> "securityDefinitions": [](start = 1, length = 23)

Is there a reference to this definition? I can't seem to find it.

Reply:
It is not a definition, it's used to define all security schemes supported by the API